backend/app/utils.py

```diff
@@ -38,17 +52,24 @@ def is_running_on_lambda():
 
 
 def get_bedrock_client(region=BEDROCK_REGION):
-    client = boto3.client("bedrock", region)
+    client = boto3.client("bedrock", region_name=region)
     return client
 
 
 def get_bedrock_runtime_client(region=BEDROCK_REGION):
-    client = boto3.client("bedrock-runtime", region)
-    return client
+    if ENABLE_BEDROCK_CROSS_REGION_INFERENCE and is_region_supported_for_cross_inference(region):
```

Comment:
Nest structure (conditions) can be simplified

Reply:
Simplified nest structure in https://github.com/aws-samples/bedrock-claude-chat/pull/571/commits/85e83810ce466b39ed6064d60f96e8be1ba37032